index.js

```diff
@@ -242,8 +333,22 @@ const addKoombeaStyles = () => {
     addStyle(css);
     return;
   }
-};
+}
 
 window.addEventListener('load', () => {
   addKoombeaStyles();
+  setInterval(function(){ // Converts booking % to h/d
+      var bookings = document.querySelectorAll('.GMGanttFixedIn, .GMGanttLeftIn, .GMGanttRightIn, .GMGanttBaseIn');
+      bookings.forEach(function(booking){
+          // Validates that the booking wasn't already converted and that it contains the % value
+          if(booking.innerHTML.split('h/d')[0] == booking.innerHTML && booking.innerHTML != '' && booking.innerHTML.match(/([0-9]*\.?[0-9]+)\s%/) != null){
+              var hours_per_day = Math.round((booking.innerHTML.match(/([0-9]*\.?[0-9]+)\s%/)[1]*8/100 + Number.EPSILON) * 100) / 100;
+              if(booking.innerHTML.match(/([0-9]*\.?[0-9]+)\s%/)[0] == booking.textContent) { // For allocation percentage bubbles at the top
+                  booking.innerHTML = hours_per_day + 'h/d';
+              } else { // For bookings
+                  booking.innerHTML = hours_per_day + 'h/d ' + booking.innerHTML;
+              }
+          }
+      });
+  },1000);
```

Comment:
You can create the function outside of the load event.
Also, I made a couple of spacing changes to the function and replaced `var` with `const` to make it use more "modern" javascript. Let me know if it doesn't work so we can use the first proposal.

```
/**
 * Converts booking % to h/d
 */
function convertBookingPercentToHD() {
  const bookings = document.querySelectorAll('.GMGanttFixedIn, .GMGanttLeftIn, .GMGanttRightIn, .GMGanttBaseIn');
  bookings.forEach((booking) => {
    // Validates that the booking wasn't already converted and that it contains the % value
    const bookingHTML = booking.innerHTML;
    if (
      bookingHTML.split('h/d')[0] == bookingHTML &&
      bookingHTML != '' &&
      bookingHTML.match(/([0-9]*\.?[0-9]+)\s%/) != null
    ) {
      const hours_per_day = Math.round(((bookingHTML.match(/([0-9]*\.?[0-9]+)\s%/)[1] * 8) / 100 + Number.EPSILON) * 100) / 100;
      if (bookingHTML.match(/([0-9]*\.?[0-9]+)\s%/)[0] == booking.textContent) {
        // For allocation percentage bubbles at the top
        booking.innerHTML = hours_per_day + 'h/d';
      } else {
        // For bookings
        booking.innerHTML = hours_per_day + 'h/d ' + bookingHTML;
      }
    }
  });
}
```

```suggestion
  setInterval(convertBookingPercentToHD, 1000);
```